test/conversations.spec.js

```diff
@@ -0,0 +1,193 @@
+const describe = require('mocha').describe;
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const client = require('twilio')('ACxxx', 'fake_token');
+const conversations = require('../src/conversations/conversations');
+
```

Comment:
To use `sinon` and `chai`, you have to tell chai to use the `sinon-chai` library:
```javascript
const sinonChai = require("sinon-chai");
const expect = chai.expect;
chai.use(sinonChai);
```
Maybe that's why the tests always pass regardless of using `calledOnce` or `calledTwice`?

Reply:
done! now it works. Thanks @mrcrow85